bot/exts/info/source.py

```diff
@@ -19,18 +19,31 @@ class BotSource(commands.Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
-    @commands.command(name="source", aliases=("src",))
-    async def source_command(self, ctx: commands.Context, *, source_item: SourceConverter = None) -> None:
+    @app_commands.command(name="source")
+    async def source_command(
+        self,
+        interaction: Interaction,
+        *,
+        source_item: app_commands.Transform[SourceType, SourceTrasnformer] = None
```

Comment:
This parameter doesn't suits in the UI well, please suggest a new name.

Reply:
Maybe `target_utility` ?

Reply:
Updated in https://github.com/python-discord/bot/pull/2446/commits/525fb54aed18526ed54e995e6e5bab6721c49baf